### PR TITLE
Mob rarity fixes

### PR DESCRIPTION
**Pull request by funkychicken493**
Mob rarities post merge:

- [x] Cowardly doesn't work, the mob ignores the walk command
- [x] Spinning needs to have the interval decreased
- [x] Confusing doesn't work
- [x] Splitting doesn't work
- [x] Sandy should send a title instead of a chat message
- [x] Sandy should create sand particle effects
- [x] Breaking doesn't actually break anything, you can still use a cool-downed weapon

### CODE REVIEW

**Comment by funkychicken493** [comment]
Effects:
- [x] Teleporting suffix sounds and particles

File: denizen_scripts/HeroCraft/mobs/suffixes/teleporting.dsc [hunk 1]
```diff
@@ -2,5 +2,8 @@ custom_mob_suffix_teleporting:
   Type: world
   debug: false
   events:
-    after entity_flagged:teleporting targets entity:
-        - teleport <context.entity> <context.target.location> if:<util.random_chance[25]>
\ No newline at end of file
+    after entity_flagged:teleporting targets entity chance:25:
+        - playsound <context.target.location> sound:entity_enderman_teleport volume:2.0
+        - playeffect <context.target.eye_location> effect:item_crack quantity:100 offset:0.5,0.5,0.5 special_data::ender_eye
+        - playeffect <context.entity.eye_location> effect:item_crack quantity:100 offset:0.5,0.5,0.5 special_data::ender_eye
```
Comment (Ph4i1ur3) [inline denizen_scripts/HeroCraft/mobs/suffixes/teleporting.dsc:8]: Why double?

Comment (funkychicken493) [reply denizen_scripts/HeroCraft/mobs/suffixes/teleporting.dsc:8]: one at target location and one at origin

**Comment by funkychicken493** [comment]
Moved to #418

**Pull request closed without merging**